pkg/pillar/hypervisor/containerd.go

```diff
@@ -49,6 +50,15 @@ func newContainerd() Hypervisor {
 	}
 }
 
+// CountMemOverhead - returns the memory overhead for a domain.
+// This implementation is used for Xen as well
+func (ctx ctrdContext) CountMemOverhead(domainName string, domainUUID uuid.UUID, domainRAMMemory int64, vmmMaxMem int64,
```

Comment:
I don't know how important this is, but as was reading the caller I saw that the order of in64 arguments is somewhat ad-hoc hence hard to remember:
1. memory
2. max memory
3. max cpus
4. cpus

It is sad that golang has no way to name arguments (unlike python) meaning that if you want named args you have to do a struct akin to
 foo(x, y, fooArgs{arg1: 27, arg2: 93})

Given the small number of callers of this function I don't propose that, but I like to complain about golang making things hard ;-)